scripts/git-checks.sh

```diff
@@ -73,7 +74,10 @@ done
 for rev in $(git rev-list $oldrev..$newrev); do
 	comparison=$(git rev-parse --quiet --verify $rev^ || true)
 
-	whitespace=$(git diff --check $comparison $rev || true)
+    # ignore changes in developers.libra.org directory
+	whitespace=$(git diff --check $comparison $rev \
+	    -- . ':!developers.libra.org' || true)
```

Comment:
@bmwill Same here re: whitespace in the website files. They have a different criteria than other files in the codebase.